model/testing/src/icon4py/model/testing/helpers.py

```diff
@@ -66,6 +71,21 @@ def allocate_data(backend, input_data):
     return input_data
 
 
+def _match_marker(marker, backend):
+    for m in marker:
+        match m.markname:
+            case "embedded_remap_error" if is_embedded(backend):
+                pytest.xfail("Embedded backend currently fails in remap function.")
+            case "embedded_as_offset_error" if is_embedded(backend):
+                pytest.xfail("Embedded backend does not support as_offset.")
+            case "levels_plus_one" if is_embedded(backend):
+                pytest.xfail("Embdeed backend does not support larger boundaries than field sizes.")
+            case "domain_dims_mismatch" if is_embedded(backend):
+                pytest.xfail("Stencil does not support missing neighbors.")
```

Comment:
Marker name and xfail message seem unrelated. Please comment what this marker is about.

Reply:
To my understanding both marked as `domain_dims_mismatch` should be tagged with both `embedded_remap_error` and `embedded_as_offset_error`. They fail in `theta_v(C2E2C[0])` for `truly_horizontal_diffusion_nabla_of_theta_over_steep_points.py:32`, which is the problem of https://github.com/GridTools/gt4py/issues/1583 but with concrete neighbor, but after that would be fixed, it would still need an `as_offset`.